routemaster/tests/test_cron.py

```diff
@@ -181,42 +182,11 @@ def processor(_app, _state, state_machine, process_wrapper):
     job, = scheduler.jobs
     job.run()
 
-    mock_trigger_gate.assert_called_with(app, gate, state_machine, mock.ANY)
-
-    assert items_to_process == ['should_not_process']
-
-
-@freezegun.freeze_time('2018-01-01 12:00')
-def test_cron_job_process_all_items_even_if_one_errors(custom_app_config):
```

Comment:
I think we still want a test with this intent, even though the implementation has changed a bit. Maybe it becomes `test_cron_processes_future_jobs_even_when_one_errors`?